content/administration/upgrade.rst

```diff
@@ -1,264 +1,340 @@
-via the `Odoo Support page <https://www.odoo.com/help>`_.
+         $ python <(curl -s https://upgrade.odoo.com/upgrade) test -d <your db name> -t <target version>
 
-Under the *Ticket Description* section, select *An issue related to my upgrade* ticket type.
+      The following command can be used to display the general help and the main commands:
 
-   .. image:: upgrade/test-assistance.png
-      :align: center
-      :alt: Selection of "An issue related to my upgrade" as Ticket Type in the support form on Odoo
+      .. code-block:: console
 
-   .. warning::
-      If you choose another *Ticket Description* type, the request will be redirected to another
-      team. This will slow down the processing and response time.
+         $ python <(curl -s https://upgrade.odoo.com/upgrade) --help
 
-Please provide as much detail as you can (i.e., videos and screenshots to illustrate your issue).
-This will avoid clarifying questions and speed up the resolution process significantly.
+      An upgraded test database can also be requested via the `Upgrade page
+      <https://upgrade.odoo.com/>`_.
+
+      .. note::
+         - For security reasons, only the person who submitted the upgrade request can download it.
+         - For storage reasons, the database's copy is submitted without a filestore to the upgrade
+           server. Therefore, the upgraded database does not contain the production filestore.
+         - Before restoring the upgraded database, its filestore must be merged with the production
+           filestore to be able to perform tests in the same conditions as it would be in the new
+           version.
+         - The upgraded database contains:
+
+           - A `dump.sql` file containing the upgraded database
+           - A `filestore` folder containing files extracted from in-database records into
+             attachments (if there are any) and new standard Odoo files from the targeted Odoo
+             version (e.g., new images, icons, payment provider's logos, etc.). This folder be
+             merged with the production filestore to get the full upgraded filestore.
 
 .. note::
-   * The purpose of the test phase is not to correct existing data or configurations in your
-     database.
-   * |assistance-contact|
+   You can request multiple test databases if you wish to test an upgrade more than once.
 
-.. _upgrade/steps-production:
+In all cases, the process is the same:
 
-The production launch
-=====================
+#. Exporting the database to a file
+#. Uploading the file to the upgrade server
+#. Running a series of :ref:`standard migration scripts <reference/upgrade/migration-scripts>` to
+   upgrade standard modules
+#. Downloading the upgraded database
+#. Importing the file into a database
+#. (*Optional*) Running a series of :ref:`custom migration scripts
+   <reference/upgrade/migration-scripts>` developed by third parties to upgrade custom modules
```

Comment:
I don't think it's clear when, why or what runs

Reply:
It is a numbered list, so the when is : after the previous step
I am not sure that "why" is important, but I can add "to upgrade them" (see full sentence below)
As for the what, yes I agree

full sentence : ```#. (*If applicable*) :ref:`Custom migration scripts <reference/upgrade/migration-scripts>`   developed by the maintainer of your custom modules are ran to upgrade them.```

What do you think ?